curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -580,7 +580,7 @@ class AsyncPrefetchCallback {
 
         int ret = s3Client_->GetDiskCacheManager()->WriteReadDirect(
             context->key, context->buf, context->len);
-        LOG_IF(ERROR, ret < 0)
+        LOG_IF(INFO, ret < 0)
```

Comment:
This log is too much, it is meaningless to change to info level, it is recommended to change to vlog

Reply:
fix